src/gatsby/src/components/park/contact.js

```diff
@@ -81,7 +81,6 @@ export default function Contact({ contact, parkContacts, operations }) {
     operations.hasPicnicShelterReservations ||
     operations.hasFrontcountryReservations ||
     operations.hasFrontcountryCabinReservations ||
-    operations.hasBackcountryPermits ||
```

Comment:
We should confirm with Amanda that this was actually a bug and not an error in the requirements.

Reply:
Good catch! Confirmed with Amanda and Brett: https://bcparksdigital.atlassian.net/browse/CMS-260